src/server.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python
+
+"""Aggregates all the Flask blueprints to initialize the API."""
+import src.config as config
+
+# Initialize the API via the spec.
+config.app.add_api("workload-api-spec.yaml")
+config.app.add_api("tokengen-api-spec.yaml")
+
+if __name__ == "__main__":
+    """Initialize a development server."""
+    config.app.run(port=8080, debug=True)
```

Comment:
Can I take that as a separate PR? That would require a bit of refactor as I'll have to understand the connexion routing otherwise I'll run into circular import problems again.

Reply:
Ok

Reply:
Turns out I had to move it anyway for gunicorn to work properly. This is now fixed.